libra_toolbox/neutronics/neutron_source.py

```diff
@@ -0,0 +1,66 @@
+# building the MIT-VaultLab neutron generator
+# angular and energy distribution
+
+from pathlib import Path
+import numpy as np
+import h5py
+import openmc
+
+
+def neutron_source_diamond(center=(0, 0, 0), reference_uvw=(0, 0, 1)):
```

Comment:
Could you also document the return type of this function? I believe it returns a list of `openmc.Source` objects?

Reply:
added return type